lib/DxilValidation/DxilValidation.cpp

```diff
@@ -970,6 +970,169 @@ static void ValidateImmOperandForMathDxilOp(CallInst *CI, DXIL::OpCode opcode,
   }
 }
 
+static bool CheckFromRegisterInterpretations(uint32_t input) {
```

Comment:
We should conform to coding standards.
```suggestion
static bool CheckFromRegisterInterpretations(uint32_t Input) {
```
Same with others below, `Input`, `OpCode`.